docs/v0.6/en/user-guide/client-libraries/go.md

```diff
@@ -36,153 +37,256 @@ import (
 {template ingester-lib-connect%
 
 ```go
-options := []grpc.DialOption{
-    grpc.WithTransportCredentials(insecure.NewCredentials()),
-}
-// To connect a database that needs authentication, for example, those on Greptime Cloud,
-// `Username` and `Password` are needed when connecting to a database that requires authentication.
-// Leave the two fields empty if connecting a database without authentication.
 cfg := greptime.NewCfg("127.0.0.1").
-    WithDatabase("public").      // change to your real database
-    WithPort(4001).              // default port
-    WithAuth("username", "password").            // `Username` and `Password`
-    WithDialOptions(options...). // specify your gRPC dail options
-    WithCallOptions()            // specify your gRPC call options
+    // change the database name to your database name
+    WithDatabase("public").
+    // default port
+    WithPort(4001).
+    // set authentication information
+    WithAuth("username", "password")
+
+client, _ := client.New(cfg)
+```
+%}
+
+{template low-level-object%
+
+```go
+// Construct the table schema for CPU metrics
+cpuMetric, err := table.New("cpu_metric")
+if err != nil {
+    // Handle error appropriately
+}
+
+// Add a 'Tag' column for host identifiers
+cpuMetric.AddTagColumn("host", types.STRING)
+// Add a 'Timestamp' column for recording the time of data collection
+cpuMetric.AddTimestampColumn("ts", types.TIMESTAMP_MILLISECOND)
+// Add 'Field' columns for user and system CPU usage measurements
+cpuMetric.AddFieldColumn("cpu_user", types.FLOAT)
+cpuMetric.AddFieldColumn("cpu_sys", types.FLOAT)
+
+// Insert example data
+// NOTE: The arguments must be in the same order as the columns in the defined schema: host, ts, cpu_user, cpu_sys
+err = cpuMetric.AddRow("127.0.0.1", time.Now(), 0.1, 0.12)
```

Comment:
Is it better to call `AddRow` multiple times in sample code?

Reply:
‌‌‌+1, avoid users misunderstanding that one `table` only can adds one row.